### PR TITLE
Blacklist has been created and ready to run

### PR DESCRIPTION
**Pull request by zeroject**
It features an http api where people can get the blacklist from a user and add to that list aswell as removing from the blacklist.
Messaging is also included as a bonus if it wants to be used by anyone.

### CODE REVIEW

File: Blacklist/Blacklist/BlackDic.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Blacklist
+{
+    public static class BlackDic
```
Comment (patrickstolc) [inline Blacklist/Blacklist/BlackDic.cs:3]: Typo?

File: Blacklist/Blacklist/Program.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+builder.Services.AddControllers();
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+builder.Services.AddSingleton(new MessageClient(RabbitHutch.CreateBus("host=rabbitmq;port=5672;virtualHost=/;username=guest;password=guest")));
```
Comment (patrickstolc) [inline Blacklist/Blacklist/Program.cs]: (╥﹏╥)

File: Blacklist/Blacklist/BlackDic.cs (hunk 1 above)
Comment (zeroject) [reply Blacklist/Blacklist/BlackDic.cs:3]: my bad will fix

**Review by patrickstolc — COMMENTED** [review]
Also missing Newtonsoft.Json?

File: Blacklist/Blacklist/Program.cs (hunk 2 above)
Comment (zeroject) [reply Blacklist/Blacklist/Program.cs]: moved to appsettings

**Pull request merged**